examples/simple-game-server/Dockerfile.windows

```diff
@@ -16,16 +16,21 @@
 # Use `--build-arg WINDOWS_VERSION=` to select the correct base image
 # See https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility
 # for details.
-ARG WINDOWS_VERSION=ltsc2019
 
-FROM mcr.microsoft.com/windows/servercore:${WINDOWS_VERSION}
+# Build Stage
+FROM --platform=linux/amd64 golang:1.20.4 as base
 
-# Unlike the Linux image the binary is built on the host since buildx cannot
-# invoke RUN commands for Windows Containers and multi-stage images
-# cannot copy data between OSes.
-COPY ["bin/simple-game-server.exe", "/server/simple-game-server.exe"]
+WORKDIR /app
+COPY go.mod go.sum ./
```

Comment:
Since we're running from root, I expect this will need to be very similar to what we did in https://github.com/googleforgames/agones/blob/main/examples/autoscaler-webhook/Dockerfile 

Which coincidentally, is basically the same as what the Linux simple-game-server Dockerfile does:
https://github.com/googleforgames/agones/blob/main/examples/simple-game-server/Dockerfile

Reply:
Modified this file after comparing Dockerfiles of autoscaler-webhook, simple-game-server, and xonotic images, but the [build](https://pantheon.corp.google.com/cloud-build/builds;region=global/29fd3d17-40e7-46a1-bbe7-8006707d91c3?mods=logs_tg_prod&project=agones-mangalpalli) failed with an error,
```
error: failed to solve: process "powershell -command GOOS=windows GOARCH=amd64 CGO_ENABLED=0 go build -o bin/simple-game-server.exe main.go" did not complete successfully: exit code: 1
```